README.md

```diff
@@ -4,12 +4,18 @@
 
 #### School Year
 
-### 2020 - 2021
+### 2021 - 2022
 
 - **Ying Yang** - Product Manager
 - **Jessica Feng** - Technical Project Manager
-- **Sheri Guo** - Product Designer
+- **Sarah Mirae Kim** - Associate Product Manager
 - **Emma Shen** - Product Designer
 - **Katrina Wang** - Software Developer
 - **Rohan Maheshwari** - Software Developer
 - **Zak Kent** - Software Developer
+- **Sydney Tran** - Software Developer
+- **Sarah Young** - Software Developer
+- **Vishok Jonnalagadda** - Software Developer 
+
+
```

Comment:
After this year's list, can you add back in the 2020-2021 members? So just copy/paste the year + list from the original ReadME.